provider/data/timezone/fingerprints.csv

```diff
@@ -1,4 +1,4 @@
-time_zone/bcp47_to_iana@1, <singleton>, 7630B, 7609B, 7efd2629d86e0cdc
-time_zone/iana_to_bcp47@3, <singleton>, 9551B, 9509B, ef97860b5f688e6d
+time_zone/bcp47_to_iana@1, <singleton>, 7622B, 7600B, 35e1d2b320b41234
+time_zone/iana_to_bcp47@3, <singleton>, 9543B, 9500B, 7218ae639c063764
```

Comment:
Nit (optional): It's not clear whether these sizes include the checksum. I think they should, at least for postcard.

Reply:
They don't, postcard doesn't track indexing overhead either, so I don't think we need to track checksizes